demo/GlutConfig.cmake

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required (VERSION 3.1)
+
+#Configure GLUT
+set(FREEGLUT_BUILD_DEMOS OFF CACHE BOOL "Build FreeGLUT demos.")
+set(FREEGLUT_BUILD_SHARED_LIBS OFF CACHE BOOL "Build FreeGLUT shared library.")
+set(FREEGLUT_PRINT_ERRORS OFF CACHE BOOL "Lib prints errors to stderr")
+set(FREEGLUT_PRINT_WARNINGS OFF CACHE BOOL "Lib prints warnings to stderr")
+
+#Set GLUT variables
+set(GLUT_INCLUDE_DIRS
+    ${GLUT_SOURCE_DIR}/include
```

Comment:
This file should be self-contained and not rely on external variables. All variables required for `add_subdirectory` shall be specified here and only rely on some global `PEGASUS_ROOT` variable